tests/test_tradesimulation.py

```diff
@@ -41,7 +49,7 @@ def test_minutely_emissions_generate_performance_stats_for_last_day(self):
                                                       data_frequency='minute',
                                                       emission_rate='minute')
         algo = NoopAlgorithm(sim_params=params)
-        algo.run(source=[], overwrite_sim_params=False)
+        algo.run(data_portal=FakeDataPortal())
```

Comment:
we could, but I was just sticking with what was being tested (ie, no trades).  Can add later if needed.